docs/csharp/language-reference/builtin-types/reference-types.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "Builtin reference types - C# reference"
+description: "Learn about reference types that have C# keywords you can use to declare them."
+ms.date: 06/25/2019
+f1_keywords: 
+  - "object_CSharpKeyword"
+  - "object"
+  - "delegate_CSharpKeyword"
+  - "delegate"
+  - "CS0123"
+  - "dynamic_CSharpKeyword"
+  - "string"
+  - "string_CSharpKeyword"
+helpviewer_keywords: 
+  - "object keyword [C#]"
+  - "delegate keyword [C#]"
+  - "function pointers [C#]"
+  - "dynamic [C#]"
+  - "dynamic keyword [C#]"
+  - "strings [C#], reference"
+  - "@ string literal"
+  - "string literals [C#]"
+  - "string keyword [C#]"
+---
+
+# Built-in reference types
+
+A number of reference types are built-in in C#. They have keywords or operators that are synonyms for a type in the .NET library. 
+
+## The object type
+
+The `object` type is an alias for <xref:System.Object?displayProperty=nameWithType> in .NET. In the unified type system of C#, all types, predefined and user-defined, reference types and value types, inherit directly or indirectly from <xref:System.Object?displayProperty=nameWithType>. You can assign values of any type to variables of type `object`. When a variable of a value type is converted to object, it is said to be *boxed*. When a variable of type object is converted to a value type, it is said to be *unboxed*. For more information, see [Boxing and Unboxing](../../programming-guide/types/boxing-and-unboxing.md). Any `object` variable can be assigned to its default value using the literal `null`.
+
+## The dynamic type
```

Comment:
If the sections to be sorted by the frequency of the type use, should this one be the last in this overview? Might be a little bit confusing to learn about bypassing of compile-time checks before getting known with strings.

Reply:
Hmm. Good idea. (I'd put `dynamic` after `object` because `dynamic` is "`object` with some metadata...) 

I like your organization better.